ui/admin/src/app/records/records.service.ts

```diff
@@ -21,10 +27,13 @@ export class RecordsService {
     );
   }
 
-
-  getRecord(record_type: string, pid: string, resolve = 0) {
```

Comment:
please leave the resolve param

Reply:
reintroduction of the parameter and update user service